pkgs/servers/mastodon/source.nix

```diff
@@ -1,11 +1,16 @@
 # This file was generated by pkgs.mastodon.updateScript.
-{ fetchgit, applyPatches }: let
+{ fetchgit, applyPatches, fetchpatch }: let
   src = fetchgit {
     url = "https://github.com/mastodon/mastodon.git";
     rev = "v4.1.1";
     sha256 = "1c0mxz45pkgmyw81z025n1ps1dkdq92337h7sd865w2fbgmzg50l";
   };
 in applyPatches {
   inherit src;
-  patches = [];
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/mastodon/mastodon/commit/61b069b20ef6977703721ae44ae8a68be6bb0992.patch";
```

Comment:
Could you please use https://github.com/mastodon/mastodon/commit/e7b81d7d9625893b1323e12215a2a98c0f19f58f ?
It's the same commit, but the one merged into `main`, so it's easy to see in which release it got included. Makes removing it later easier.

Reply:
Fixed.